bodhi/server/util.py

```diff
@@ -53,8 +55,16 @@ def header(x):
 
 
 def get_rpm_header(nvr, tries=0):
-    """ Get the rpm header for a given build """
+    """
+    Get the rpm header for a given build.
 
+    Args:
+        nvr (basestring): The name-version-release string of the build you want headers for.
```

Comment:
Do you really mean basestring? This will get wonky when you want to use Python 3.

I often use ``six.text_type`` and ``six.binary_type`` so it's clear what I mean and also note in the comment whether or not it must be unicode.

Reply:
I'm kind of on the fence about ```basestring``` vs. ```six.text_type```. I don't intend to support Python 2 and Python 3 at the same time server-side whenever I do switch to Python 3, so I'll probably just do a search and replace on basestring at that time.

I certainly don't want to audit, because even if it *is* being used with only one of ```str``` or ```unicode```, that doesn't mean it couldn't work with the other. I mostly want to express the "duck type" that the function expects to be able to work with.

So I think I'll leave it as-is for now, since "basestring" is unique enough to be easily sed'able later when we actually start supporting Python 3.